src/stores/user-store.js

```diff
@@ -12,10 +12,11 @@ class UserStore {
   }
 
   async findUserByLogin({ provider, emailOrUsername }, { session } = {}) {
+    const lowerCasedEmailOrUsername = emailOrUsername?.toLowerCase() || '';
```

Comment:
I realized this function has much more logic than the store should have (also with the `switch` below). I left it for the fix, but will create a ticket to put the decision logic into the service.

Reply:
That's true, the best match decision does not belong here. Actually this method should rather be "getUserByEmailOrUsername" .

Reply:
Oh wait, but for username we allow upper case 🤔

Reply:
:+1:

Reply:
Don't wait, I see you only used it for email 😆 sorry!